h2o-core/src/main/java/water/TypeMap.java

```diff
@@ -65,11 +67,11 @@
 
   };
   // Class name -> ID mapping
-  static private final NonBlockingHashMap<String, Integer> MAP = new NonBlockingHashMap<>();
+  public static final NonBlockingHashMap<String, Integer> MAP = new NonBlockingHashMap<>();
```

Comment:
oops! These can go back to "static private" as before, any way to revert these changes so I don't hurt the history?